meshtastic/interfaces/ble/gating.py

```
@@ -0,0 +1,405 @@
+def is_currently_connected_elsewhere(
+    addr: Optional[str], owner: Optional[Any] = None
+) -> bool:
+    """
+    Determine whether a BLE address is recorded as connected by a different owner.
+
+    Normalizes the provided address via addr_key; if normalization yields None the function returns False. While checking, stale records are pruned when the recorded owner has been garbage-collected, reports disconnected, or an unowned claim has expired.
+
+    Parameters
+    ----------
+        addr (Optional[str]): BLE address to check; will be normalized via addr_key.
+        owner (Optional[Any]): Caller’s owner instance; if it matches the recorded owner by identity or recorded id, the address is not considered connected elsewhere.
```

Ruff (0.15.1)

[warning] Docstring contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF002)